content/en/docs/ops/diagnostic-tools/istioctl/index.md

```diff
@@ -162,7 +166,11 @@ $ source ~/istioctl.bash
 
 Installing the ZSH auto-completion file
 
-For ZSH users, the `istioctl` auto-completion file is located in the `tools` directory. Copy the `_istioctl` file to your home directory, or any directory of your choosing (update directory in script snippet below), and source the `istioctl` auto-completion file in your `.zshrc` file as follows:
+For ZSH users, the `istioctl` auto-completion file is located in the `tools` directory.
```

Comment:
The file has an initial underscore, `_istioctl`.

Reply:
Fixed it and also in the bash instructions (`.bash` extension was missing)